discovery/discovery.yaml

```diff
@@ -131,7 +131,7 @@ components:
           schema:
             $ref: '#/components/schemas/serviceinstance'
   responses:
-    servicespostresponse:
+    servicesputresponse:
```

Comment:
I think this one needs to remain `servicespostresponse`

Reply:
fixed